### DISCUSSION TITLE
Celery Latency

### DISCUSSION THREAD

**Discussion by Angeschossen** [WeblateOrg/weblate#11126] [Q&A]
I'm running Weblate in a docker container. I've followed the installation instructions from the docs and have noticed that the site is initially very slow, and Celery Latency is sometimes about 200-2000ms in the performance tab. It seems to have something to do with Celery. No other warnings or issues are listed in the performance tab. Does anyone have any idea or experience with this?

Reloading the page multiple times make the Celery latency go down to a few ms. Database and cache are always 0-2ms.

If you browse the page for a bit and change pages etc. the celery latency will go up again randomly and then down after being really high for once.

**Comment by nijel** [comment]
Is there anything visible in the docker logs?

**Reply by Angeschossen** [reply]
Thank you for your reply. Turns out it was just my VPS. The provider was overselling their machines. No issues with weblate. Thank you!
